src/components/MoneyRequestConfirmationList.js

```diff
@@ -104,7 +104,7 @@ class MoneyRequestConfirmationList extends Component {
      */
     getSplitOrRequestOptions() {
         return [{
-            text: this.props.translate(this.props.hasMultipleParticipants ? 'iou.split' : 'iou.request', {
+            text: this.props.translate(this.props.hasMultipleParticipants ? 'iou.splitAmount' : 'iou.requestAmount', {
```

Comment:
A minor change that was necessary. Else the new 'split' string would need an alternative name.